cfalertdialog/src/main/java/com/crowdfireapp/cfalertdialog/CFAlertDialog.java

```diff
@@ -53,38 +89,131 @@ private CFAlertDialog(Context context, int theme) {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+        // Inflate the view
         LayoutInflater layoutInflater = (LayoutInflater) getContext().getSystemService(Context.LAYOUT_INFLATER_SERVICE);
-        View view = layoutInflater.inflate(R.layout.crowdfire_dialog_layout, null);
+        View view = layoutInflater.inflate(R.layout.cfalert_layout, null);
```

Comment:
We have no provision to set custom typeface to anything in the dialog...the title, message, buttons etc.

Reply:
Our preset UI is the default alert with customised text message and buttons. For finer control over the elements we have provision for User to use a custom view/layout in the alert.

Reply:
But for a simple dialog with just a title and message, there is no way for me to have the dialog align with the app style

Reply:
We can may be add support for this in the upcoming updates.